libs/template/templates/dbt-sql/template/{{.project_name}}/resources/{{.project_name}}_job.yml.tmpl

```diff
@@ -3,10 +3,10 @@ resources:
     {{.project_name}}_job:
       name: {{.project_name}}_job
 
-      schedule:
-        # Run every day at 9:27 AM
-        quartz_cron_expression: 21 27 9 * * ?
-        timezone_id: UTC
+      trigger:
+        periodic:
```

Comment:
Could you retain the comment saying this runs once per day, perhaps with a link/comment to docs?

Reply:
Ideally we have a DABs page to link to, which then itself links to the API ref and applicable conceptual/UI articles (because we don't put YAML in the UI or conceptual docs). But since we don't have that DABs page ... should we? It's really hard to know how much info and YAML examples we need for these fields in the DABs docs that can easily be determined by looking at the API ref and other conceptual material.  And we really don't want to be duplicating this info as that is hard to maintain ... 

I'm torn whether linking to https://docs.databricks.com/api/workspace/jobs/create#trigger (the anchor I think would be great - mapping the yaml and payload field is something I think we need to enforce a lot) or https://docs.databricks.com/en/jobs/schedule-jobs.html would be better. Probably the former if I have to choose!

Reply:
I don't think we need a DAB-specific page for this specifically. I was hoping for a page explaining the guarantees of periodic triggers. For example, does it run at the same time every day, or can it run at 1am on the first day, and 11pm on the second day. If there is some quick one-liner we can inline that works too.

This is clear (by definition) for cron expressions.

Reply:
https://docs.databricks.com/api/workspace/jobs/create#trigger with the anchor seems like the best we have right now. @juliacrawf-db maybe you can review whether it's worthwhile to change the other doc? I'd say we can just point to the anchor for now.

For me, the real value of the comment is probably just to indicate that there's something "live" here, which can incur cost and may require the customer's intention. Otherwise the syntax speaks mostly for itself. The main thing the user may need from docs to know is when and how to switch to cron syntax.

Reply:
Can you clarify what the guarantee is though?

Will it run at the same random time each day, or is this randomized every day?

Reply:
Based on the API doc info, for periodic, I think the guarantee is that it runs at exactly that specified delta from the time the last run happened. And the first run is initiated by the UI or API request. 

So I think pointing to the API doc (with the trigger anchor) is good, but the comment could also be modified to clarify: "Run this job exactly one day from the last run."  And I submitted a [small PR](https://github.com/databricks-eng/docs/pull/17900) to add this to the jobs docs.

Reply:
> Based on the API doc info, for periodic, I think the guarantee is that it runs at exactly that specified delta from the time the last run happened. And the first run is initiated by the UI or API request.

Yeah, exactly this.

> So I think pointing to the API doc (with the trigger anchor) is good, but the comment could also be modified to clarify: "Run this job exactly one day from the last run." And I submitted a [small PR](https://github.com/databricks-eng/docs/pull/17900) to add this to the jobs docs.

How about "Run this job every day, exactly one day from the last run"

Reply:
Oh good point, the repeat bit is missing from my suggestion. Your version is great!

Reply:
Updated.